src/components/ImportFile/ImportFile.test.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { mount, shallow } from 'enzyme';
+import { ImportFile, ImportFileContent } from './index';
+
+const files = [];
+
+test('ImportFile is working properly', () => {
+  const component = shallow(
+    <ImportFile acceptedTypes="image/png" maxAmount={4} maxSize={4000} />
+  );
+  component.instance().onDrop(files, files);
```

Comment:
It is usually a better practice to test the component as it would be used.  An example of this for `onDrop` would be to do something like. 
```javascript
component.find(Dropzone).props().onDrop(files, files)`
expect(component.find(ImportFileContent).props().state).toBe(dropStates.ACCEPTED) // or the expected state
```

Reply:
I'm getting different feedbacks on testing from others, I think ill learn more about `enzyme` before proceeding with the tests. Until now i've been told that its fine test the components with no props/children just to know they haven't changed.

Reply:
Correct sorry I definitely didn't want to contradict things here. Good to leave as is for now.